app/assets/javascripts/views/batch_actions_view.js.coffee

```diff
@@ -0,0 +1,145 @@
+#// Models the batch actions done on index pages
+class ELMO.Views.BatchActionsView extends ELMO.Views.ApplicationView
+
+  el: '#index_table'
+
+  events:
+    'click #select_all_link': 'select_all_clicked'
+    'click a.batch_op_link': 'submit_batch'
+    'click a.select_all_pages': 'select_all_pages'
+    'change input[type=checkbox].batch_op': 'checkbox_changed'
+
+  initialize: (params, search_form_view) ->
```

Comment:
Please move `search_form_view` into params.

Reply:
this is non-trivial to do, so we decided to not do it